frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/LauncherConfiguration.java

```diff
@@ -59,6 +59,12 @@
   // is also limited to 100MB = 500K * 200 bytes/task.
   private Integer maxTotalTaskNumber = 500000;
 
+  private Integer amContainerBasePort = 2000;
+  private Integer amSearchNodeBufferFactor = 2;
+
+  private Boolean amSkipLocalTriedResource = false;
+  private Boolean amAllowNoneGpuJobOnGpuNode = true;
```

Comment:
Move it between line 102 and 104.
Align with windows, rename as amContainerBasePort -> amContainerDynamicPortRange ?